assets/css/src/amp-playlist-shortcode.css

```diff
@@ -2,10 +2,6 @@
 * For the custom AMP implementation of the 'playlist' shortcode.
 */
 .wp-playlist .wp-playlist-current-item img {
-	margin-right: 0;
-}
-
-.wp-playlist .wp-playlist-current-item amp-img {
```

Comment:
Not sure what was going on here originally, but since `img` gets converted to `amp-img` the result is the `margin-right:0` would get immediately overridden by the subsequent `margin-right: 10px`. So this is just removing an overridden rule.